data/yam/agama/auto/sle_ext4.jsonnet

```diff
@@ -0,0 +1,42 @@
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE}}'
+  },
+  user: {
+    fullName: 'Bernhard M. Wiedemann',
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+    userName: 'bernhard'
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true
+  },
+  storage: {
+    drives: [
+      {
+        partitions: [
+          {
+            filesystem: { path: '/', type: 'ext4' }
+          },
+          {
+            filesystem: { path: 'swap', type: 'swap' }
+          }
```

Comment:
as discussed, swap might not be needed either, would be picked from product information, so we can drop it.

Reply:
I removed the swap and run the VRs, it seems the installation didn't add swap by default if I remove it from the jsonnet file:
![Screenshot from 2025-01-20 13-51-37](https://github.com/user-attachments/assets/cbdc160e-cd4c-4602-9749-899ec283b32b)
So do we need to add swap back?

Reply:
Please, read thoroughly the documentation provided in the ticket:
https://agama-project.github.io/docs/user/unattended/storage
```
partitions: [
        { generate: 'default' },
        { filesystem: { path: '/', type: 'ext4' } }
      ]
```

Reply:
Updated, and no issues on VRs now, thanks.